yaptide/persistence/models.py

```diff
@@ -10,6 +10,9 @@
 from yaptide.persistence.database import db
 from yaptide.utils.enums import EntityState, PlatformType
 
+# ---------- IMPORTANT ------------
```

Comment:
Make it a module level doctoring, placing as the first line of the file in following form:

```
---------- IMPORTANT ------------
Read documentation in persistency.md. It contains information about database development with flask-migrate.
```